src/IncludedCommands/UsefulFun/fly.lua

```diff
@@ -10,14 +10,31 @@ local Players = game:GetService("Players")
 local PhysicsService = game:GetService("PhysicsService")
 local RunService = game:GetService("RunService")
 local UserInputService = game:GetService("UserInputService")
+local ContextActionService = game:GetService("ContextActionService")
 
 local IncludedCommandUtil = require(script.Parent.Parent:WaitForChild("IncludedCommandUtil"))
 local Types = require(script.Parent.Parent.Parent:WaitForChild("Types"))
 
+-- Create a link for every InputBegan and InputEnded
+local MultiPlatform = {
+
+    -- Keyboard
```

Comment:
One last formatting change and it should be done: for constants, snake case with caps locks is used (ex: `MULTI_PLATFORM`) is used. I'd also like to request the name be a bit more descriptive than that, like `MULTIPLATFORM_KEYCODE_MAPPING`.

```suggestion
local MULTIPLATFORM_KEYCODE_MAPPING= {
    -- Keyboard
```